docs/get-started/quickstarts/detect-people.md

```diff
@@ -33,77 +35,76 @@ Click the name of a machine to go to that machine's page, where you'll find a va
+Now add a `ML model` **Service**.
+Click **+** and select the `ML model` type, then select the `TFLite CPU` model.
 
 In the resulting ML model service configuration pane, ensure that **Deploy model on machine** is selected for the **Deployment** field.
 
-Click on **Select models** to open a dropdown with all of the ML models available to you.
-Select the `ml-models-scuttle:people` model (the **people** model by **ml-models-scuttle**) to deploy an object detection TFLite model that has been trained to be able to detect a person.
+Click on **Select models** and select the `ml-models-scuttle:people` model (the **people** model by **ml-models-scuttle**) to deploy an object detection TFLite model that has been trained to be able to detect a person.
 
 Click the **Save** button in the top right corner of the page to save your configuration.
 
 For more detailed information, including optional attribute configuration, see the [`tflite_cpu` docs](/services/ml/deploy/tflite_cpu/).
 
 {{% /expand%}}
-{{%expand "Step 5: Configure your detector" %}}
+{{%expand "Step 5: Configure a vision service" %}}
+
+To use the deployed person detection model to detect people on a camera stream, you need another service that applies it to the camera stream.
 
-Navigate to the **CONFIGURE** tab of your machine's page in the [Viam app](https://app.viam.com).
-Click the **+** icon next to your machine part in the left-hand menu and select **Service**.
-Select the `vision` type, then select the `ML model` model.
-Enter a name or use the suggested name for your service and click **Create**.
+Add a `vision` **Service** and select the `ML model` model.
 
 Select the ML model service your person detector model is deployed on (which you created in step 4) from the **ML Model** dropdown.
-Then, click **{}** (Switch to advanced) in the top right corner of the configuration pane.
+
+{{% /expand%}}
+{{%expand "Step 6: Control person detection" %}}
+
+Now, test your person detection on in the **Test** section of the computer vision service's configuration panel or on the **CONTROL** tab.
+
+You will see your camera stream and see detections as labeled boxes on the images along with position, dimension, class and confidence data.
+Detections in class `Person` with a high confidence score show positive person detections, but the ML model can also detect other objects:
+
+{{<imgproc src="/get-started/quickstarts/vision-card-more-detections.png" resize="x1100" declaredimensions=true alt="Positive person detection on the vision card with a lower default minimum confidence threshold." >}}
+
+{{% /expand%}}
+{{%expand "(Optional) Step 7: Limit the number of detections" %}}
+
+If you are seeing a lot of detections, you can set a minimum confidence threshold.
+
+On the configuration page of the vision service in the top right corner, click **{}** (Switch to advanced).
 Add the following JSON to the JSON configuration to set the `default_minimum_confidence` of the detector:
 
 ```json
```

Comment:
[opt] Would be great (but out of diff) to add a screenshot showing this so they can see indentation etc.

Reply:
Add full config for it